dist/ReactToastify.css

```
@@ -112,9 +112,12 @@
   height: 5px;
   z-index: 9999;
   opacity: 0.7;
-  animation: Toastify__trackProgress linear 1 forwards;
   background-color: rgba(255, 255, 255, 0.7);
   transform-origin: left; }
+  .Toastify__progress-bar--animated {
+    animation: Toastify__trackProgress linear 1 forwards; }
+  .Toastify__progress-bar--controlled {
+    transition: all .2s; }
```

I don't want to be picky but for the transition I think it's better to use `transform` instead of `all`, so we know exactly which property is impacted. What do you think ?
```suggestion
    transition: transform .2s; }
```